api/src/main/java/org/openmrs/ui/framework/UiFrameworkConstants.java

```diff
@@ -29,6 +29,8 @@
 	
 	public static final String GP_FORMATTER_JS_DATE_FORMAT = "uiframework.formatter.JSdateFormat";
 	
+	public final static String GP_TIMEZONE_CONVERSIONS = "timezone.conversions";
```

Comment:
Do you mind adding a description for this constant?

Reply:
Sure, something like ` //To user timezones this GP should be true.` or something more specific?

Reply:
As long as `user` means `use` 😊